src/rcore.c

```diff
@@ -3457,6 +3457,29 @@ void OpenURL(const char *url)
 #endif
 #if defined(PLATFORM_WEB)
         emscripten_run_script(TextFormat("window.open('%s', '_blank')", url));
+#endif
+#if defined(PLATFORM_ANDROID)
+        JNIEnv *env;
+        JavaVM *vm = CORE.Android.app->activity->vm;
```

Comment:
As per [the documentation](https://developer.android.com/training/articles/perf-jni#javavm-and-jnienv), `JNIEnv` and `JavaVM` require `jni.h`, shouldn't the include be added?

Reply:
You're right! somehow I missed it!